### DISCUSSION TITLE
How to offset the bottom axis to the middle of chart (at 0) if we have positive and negative y-values

### DISCUSSION THREAD

**Discussion by ivanleong** [patrykandpatrick/vico#887] [Questions]
### Question

Currently by default the bottom axis sits at the bottom value, can i offset it to the 0 mark?

![Screenshot 2024-09-20 at 10 39 16](https://github.com/user-attachments/assets/9b4a2613-55a5-4bbf-a454-46b90639cea3)

### Vico version(s)

2.0.0-alpha.21

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello, @ivanleong. The bottom axis cannot be offset. However, you can use [`Decoration`](https://patrykandpatrick.com/vico/wiki/cartesian-charts/decorations) to draw a line for _y_ = 0.
